### PR TITLE
VACMS-5005: Update content build to use new jobs

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

See #5005. 

## Testing done

We can't test until we deploy :(

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/sites/default/settings/settings.staging.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 include dirname(__FILE__) . '/settings.brd_common.php';
 
-$settings['jenkins_build_job_path'] = '/job/builds/job/vets-website-content-vagov' . $settings['jenkins_build_env'];
+$settings['jenkins_build_job_path'] = '/job/builds/job/content-build-content-only-vagovdev' . $settings['jenkins_build_env'];
```
Comment (timwright12) [inline docroot/sites/default/settings/settings.staging.php:7]: @indytechcook  Do you want to use vagovdev here? The prod one is `builds/content-build-content-only-vagovprod` and staging is `builds/content-build-content-only-vagovstaging`

Comment (indytechcook) [reply docroot/sites/default/settings/settings.staging.php:7]: Thanks!  I meant to only paste in `content-build-content-only-vagov`!

File: docroot/sites/default/settings/settings.dev.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 include dirname(__FILE__) . '/settings.brd_common.php';
 
-$settings['jenkins_build_job_path'] = '/job/builds/job/vets-website-content-vagov' . $settings['jenkins_build_env'];
+$settings['jenkins_build_job_path'] = '/job/builds/job/content-build-content-only-vagovdev' . $settings['jenkins_build_env'];
```
Comment (indytechcook) [inline docroot/sites/default/settings/settings.dev.php:7]:
```suggestion
$settings['jenkins_build_job_path'] = '/job/builds/job/content-build-content-only-vagov' . $settings['jenkins_build_env'];
```

File: docroot/sites/default/settings/settings.staging.php (hunk 1 above)
Comment (indytechcook) [inline docroot/sites/default/settings/settings.staging.php:7]:
```suggestion
$settings['jenkins_build_job_path'] = '/job/builds/job/content-build-content-only-vagov' . $settings['jenkins_build_env'];
```

**Comment by indytechcook** [comment]
Closing to keep all content-build separations on a single PR: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/5199

**Pull request closed without merging**